src/sentry/static/sentry/app/views/performance/realUserMonitoring/utils.tsx

```diff
@@ -3,7 +3,7 @@ import {ECharts} from 'echarts';
 
 import {HistogramData, Rectangle, Point} from './types';
 
-export function generateRealUserMonitoringRoute({orgSlug}: {orgSlug: String}): string {
+export function generateWebVitalsRoute({orgSlug}: {orgSlug: String}): string {
   return `/organizations/${orgSlug}/performance/summary/rum/`;
```

Comment:
Should this URL also change?

Reply:
How about we change it to `/organizations/${orgSlug}/performance/summary/vitals/`?

Reply:
That sounds great!